### PR TITLE
5071 publication page template

### LINKED ISSUE #5071: Publications | Publication page template exists
- New page template called Publication page
- Available as a child of any of the main wagtail pages
- Only child page available is chapter page
- Use blog page header and footer

### Acceptance criteria
This is just a basic ugly template that inherits the base template for the site, doesn't match specs. 

When a user visits a publication page, they see a basic template.

### PR DESCRIPTION
**Pull request by easherma**
Closes #5071
Related PRs/issues #

Super bare-bones implementation of publication page, to be build on in subsequent PRs. 

## Checklist

_Remove unnecessary checks_

**Tests**
- [ ] Is the code I'm adding covered by tests?

**Changes in Models:**
- [x] Did I update or add new fake data?
- [ ] Did I squash my migration?
- [x] [Are my changes backward-compatible](https://github.com/mozilla/foundation.mozilla.org/blob/master/docs/workflow.md#django-migrations-what-to-do-when-working-on-backward-incompatible-migrations). If not, did I schedule a deploy with the rest of the team?

**Documentation:**
- [x] Is my code documented?
- [x] Did I update the READMEs or wagtail documentation?

### CODE REVIEW

File: docs/local_development.md [hunk 1]
```diff
@@ -194,11 +200,13 @@ I would recommend watching [An Intro to Docker for Djangonauts](https://www.yout
 All our containers run on Linux.
 
 For local development, we have two Dockerfiles that define our images:
+
 - `Dockerfile.node`: use a node8 Debian Stretch slim base image from the Docker Hub and install node dependencies,
 - `Dockerfile.python`: use a python3.7 Debian Stretch slim base image, install required build dependencies before installing pipenv and the project dependencies.
-We don't have a custom image for running postgres and use one from the Docker Hub.
+  We don't have a custom image for running postgres and use one from the Docker Hub.
```
Comment (Pomax) [inline docs/local_development.md:206]:
```suggestion

We don't have a custom image for running postgres and use one from the Docker Hub.
```

This is normal text, not a quote =)

File: docs/stack.md [hunk 2]
```diff
@@ -10,14 +10,16 @@ CSS is generated from [Sass](http://sass-lang.com/). The [Mofo Bootstrap](https:
...
-React is used *à la carte* for isolated component instances (eg: a tab switcher) since this is not a single page application, but rather a static generated website. This precludes the need for Flux architecture, or such libraries as React Router.
+React is used _à la carte_ for isolated component instances (eg: a tab switcher) since this is not a single page application, but rather a static generated website. This precludes the need for Flux architecture, or such libraries as React Router.
 
 To add a React component, you can target a container element from `/source/js/main.js` and inject it.
 
 ### Django and Wagtail
 
 Django powers the backend of the site, and we use Wagtail with Django to provide CMS features and functionality.
 
+If you are making a new page for the site, make sure the Page inherits both the Wagtail Page as well as `FoundationMetadataPageMixin` in order for a page to display properly.
```
Comment (Pomax) [inline docs/stack.md:21]:
```suggestion
If you are defining a new page class for the site, make sure it inherits both the Wagtail `Page` class as well as the `FoundationMetadataPageMixin` mixin. The first is a general Wagtail requirement, and the latter ensures that page metadata used for SEO and the like is (generally) correct.
```

File: docs/stack.md [hunk 3]
```diff
@@ -30,14 +32,14 @@ We use [wagtail-experiments](https://github.com/torchbox/wagtail-experiments) fo
...
 
-
 ### File Structure
 
 ```
 /
 ├── dest <- Compiled code generated from source. Don't edit!
 ├── network-api <- Django site code
 │   ├── networkapi <- Django apps live within this directory
+        └── wagtailpages <- most of the pages using wagtail are here
```
Comment (Pomax) [inline docs/stack.md:42]: :+1:

File: network-api/networkapi/wagtailpages/factory/__init__.py [hunk 4]
```diff
@@ -10,7 +10,8 @@
     participate_page_featured_highlights,
     participate_page,
     styleguide,
-    youtube_regrets_page
+    youtube_regrets_page,
+    publication,
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/factory/__init__.py:14]: a nit: keeping lists like these sorted is _super_ useful for future debugging.

File: network-api/networkapi/wagtailpages/factory/__init__.py [hunk 5]
```diff
@@ -28,6 +29,7 @@ def generate(seed):
     participate_page_featured_highlights.generate(seed)
     styleguide.generate(seed)
     youtube_regrets_page.generate(seed)
+    publication.generate(seed)
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/factory/__init__.py:32]: same here, unless `publication` relies on any of the builds preceding it, let's keep it alphabetically sorted.

File: network-api/networkapi/wagtailpages/models.py [hunk 6]
```diff
@@ -96,4 +100,5 @@
     Signup,
     Styleguide,
     YoutubeRegretsPage,
+    PublicationPage,
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/models.py:103]: (also sorted =)

File: network-api/networkapi/wagtailpages/pagemodels/base.py [hunk 7]
```diff
@@ -236,43 +236,43 @@ class ParticipatePage2(PrimaryPage):
...
-          [
-            ImageChooserPanel('ctaHero'),
-            FieldPanel('ctaHeroHeader'),
-            FieldPanel('ctaHeroSubhead'),
-            FieldPanel('ctaButtonTitle'),
-            FieldPanel('ctaButtonURL'),
-          ],
-          heading="Primary CTA",
-          classname="collapsible"
+            [
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/base.py:239]: As per slack: yeah these changes should not have been necessary =D

File: network-api/networkapi/wagtailpages/pagemodels/modular.py [hunk 8]
```diff
@@ -14,7 +14,12 @@
 
 class ModularPage(FoundationMetadataPageMixin, Page):
     """
-    The base class offers universal component picking
+    This base class offers universal component picking.
+    Note: 
+    Modular pages are an artifact from a previous iteration of the codebase, 
+    where it was the top level class for all page types.
+    It's still "used" in the current codebase mostly because changing the class inheritance would be labor intensive. 
+    It's not a recomended pattern for new pages.
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/modular.py:22]:
```suggestion
    Note: this is a legacy class, see https://github.com/mozilla/foundation.mozilla.org/issues/5071#issuecomment-675720719
```

File: network-api/networkapi/wagtailpages/pagemodels/publications/publication.py [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class PublicationPage(FoundationMetadataPageMixin, Page):
+    """
+    This is the root page of a publication. From here the user can browse to the various sections (called chapters).
+    It will have information on the publication, its authors, and metadata from it's children
+    """
+
+    # TODO: once we make chapter pages, add that as a subpage type
+    subpage_types = []
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/publications/publication.py:15]: are publication pages not also their own child? (I don't actually know, but I'd assume they are)

File: network-api/networkapi/wagtailpages/pagemodels/base.py (hunk 7 above)
Comment (easherma) [reply network-api/networkapi/wagtailpages/pagemodels/base.py:239]: @Pomax is this a dealbreaker? do you want me to go back and change these lines back?

File: network-api/networkapi/wagtailpages/pagemodels/publications/publication.py (hunk 9 above)
Comment (easherma) [reply network-api/networkapi/wagtailpages/pagemodels/publications/publication.py:15]: There may not actually be many if any differences between chapter pages and publication pages(though there are some differences on the frontend). I think thats what you're getting at?

File: network-api/networkapi/wagtailpages/factory/__init__.py (hunk 4 above)
Comment (Pomax) [inline network-api/networkapi/wagtailpages/factory/__init__.py:14]: want to still move this up?

File: network-api/networkapi/wagtailpages/pagemodels/publications/publication.py (hunk 9 above)
Comment (Pomax) [reply network-api/networkapi/wagtailpages/pagemodels/publications/publication.py:14]: No, just genuine curiosity: I know that campaigns can be children of campaigns, so I was wondering if publications were supposed to be children of publications, but whether or not they do won't hold up this PR

**Pull request merged**